lib/sequel/plugins/oplog.rb

```diff
@@ -127,10 +127,6 @@ def update(*_attr)
         def insert
           raise NotImplementedError.new('You should be inserting model instances.')
         end
-
-        def delete
-          raise NotImplementedError.new('You should be *destroying* model instances.')
-        end
```

Comment:
We need unfortunately to do hard deletes.

Reply:
Could we use the same hard delete mechanism of the rollbacks rather than removing this protective layer?

Reply:
yep!